Testscripts/Windows/Run-LtpSuite.ps1

```diff
@@ -129,7 +184,8 @@ function Main {
             $CurrentTestResult.TestSummary += New-ResultSummary -testResult $currentResult -metaData $metaData `
                 -checkValues "PASS,FAIL,ABORTED" -testName $CurrentTestData.testName
         } else {
-            $resultArr += Get-SQLQueryOfLTP -currentTestResult $CurrentTestResult -currentTestData $CurrentTestData
+            $null = Get-SQLQueryOfLTP -currentTestResult $CurrentTestResult -currentTestData $CurrentTestData
+            $resultArr += "PASS"
```

Comment:
why do you change it to always PASS?

Reply:
Because the bash script returned the "TestCompleted" status.
We should only set the test as failed if the bash script is aborted.

Reply:
then the test result will never be failed, it will be either Aborted or Pass. My understanding is if there's any sub test case fails, then the test result should be failed

Reply:
We had discussed in meeting that we will need to maintain a baseline ltp results per each platform, and exclude the cases which are expected to fail. Before that is achieved, we just fail the test case if there's any sub cases fail.
I will approve this PR, and file new PRs to address this.